app/storage/blob-service-client.js

```diff
@@ -10,7 +10,7 @@ if (config.useConnectionString) {
 } else {
   console.log('Using DefaultAzureCredential for BlobServiceClient')
   const uri = `https://${config.storageAccount}.blob.core.windows.net`
-  blobServiceClient = new BlobServiceClient(uri, new DefaultAzureCredential())
+  blobServiceClient = new BlobServiceClient(uri, new DefaultAzureCredential({ managedIdentityClientId: process.env.AZURE_CLIENT_ID }))
```

Comment:
Could add new lines to the object

Reply:
Yeah probably a better way of doing this. Will need to have a think